pytorch_lightning/trainer/trainer.py

```diff
@@ -1681,7 +1681,7 @@ def accelerator(self) -> Accelerator:
         return self.training_type_plugin.accelerator
 
     @property
-    def training_type_plugin(self) -> TrainingTypePlugin:
+    def training_type_plugin(self) -> Strategy:
```

Comment:
Are we planning on completely get rid of training_type_plugin this word in code base? It still heavily used in trainer.py and accelerator_connector.py

Reply:
Yes, everywhere. It only makes sense if it is consistent everywhere.